### ISSUE TITLE
V3 jquery.event.drop

### ISSUE THREAD

**Issue by kofifus** [6pac/SlickGrid#707]
looking at the instructions for new V3 ( https://github.com/6pac/SlickGrid/wiki/Major-version-3.0----Removal-of-jQueryUI-requirement-(replaced-by-SortableJS) )

There is no mention of `jquery.event.drop` - is `jquery.event.drop-2.3.0.min.js` not needed anymore as well ?

**Comment by ghiscoding** [comment]
I don't think it was ever needed, except for a very special Example [Row selection & reordering](http://6pac.github.io/SlickGrid/examples/example9-row-reordering.html) which only uses `jquery.event.drop` for dropping a row hover the Recycle Bin button.... So in summary, no it's no needed at all (and probably never was)
